src/datajudge/constraints/base.py

```diff
@@ -113,7 +113,16 @@ class Constraint(abc.ABC):
     """
 
     def __init__(
-        self, ref: DataReference, *, ref2=None, ref_value: Any = None, name: str = None
+        self,
+        ref: DataReference,
+        *,
+        ref2=None,
+        ref_value: Any = None,
+        name: str = None,
+        output_postprocessing_sorter: Callable[
+            [Collection, Optional[Collection]], Collection
+        ] = None,
+        output_remainder_slicer=slice(5),
```

Comment:
Could the logic for this "slicer" be given as a postprocessor instead of needing an extra argument?

Reply:
Not easily, since the postprocessor is applied in more places than the slicing, there are many instances where the slicing was not present before in the library, and i did not want to introduce it there